RELEASE_NOTES.md

```diff
@@ -1,3 +1,84 @@
+## Cookstyle 5.12
+
+### 7 New Chef Cops
+
+#### ChefDeprecations/Cheffile
+
+The `ChefDeprecations/Cheffile` cop detects a `Cheffile` in cookbooks used for dependency solving with `librarian-chef`. The librarian-chef project is no longer maintained and user should handle dependency solving with Chef Infra Policyfiles of a Berkshelf Berksfile.
```

Comment:
```suggestion
The `ChefDeprecations/Cheffile` cop detects if a `Cheffile` is used by `librarian-chef` to solve dependencies. The librarian-chef project is no longer maintained and users should handle dependency solving with Chef Infra Policyfiles or a Berkshelf Berksfile.
```

Reply:
I'm not sure about this suggestion. The Cheffile is only present if you use librarian-chef. We're flagging the librarian-chef usage based on that file.

Reply:
I added the typo fixes from this